src/elements/Typography.tsx

```diff
@@ -20,8 +20,6 @@ const Typography: FunctionComponent<TypographyProps> = ({
   sx,
   ...rest
 }) => {
-  const theme = useTheme();
```

Comment:
This was throwing an error in the console and i think that we are using the theme from `src/theme/theme.ts`. @scandycuz let me know if i should put it back and continue using the `useTheme` from MUI

Reply:
They should reference the same theme. Was it throwing an error in the test? The component likely just needs to be wrapped in the theme provider for the hook to work. If you use the `renderWithContext` function from the `common/testUtils` file, instead of the `render` function from the React testing library, it should resolve the error.

Reply:
I reverted the change and ran the tests again and they seemed to run fine. There are other issues with our tests though that we can look into later on